homework.md

```diff
@@ -4,77 +4,87 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+white
 
 ## Question 3
 
-What is the  2 * 3 * 6 * 1 * 10 * 34?
+What is the 2 _ 3 _ 6 _ 1 _ 10 \* 34?
 
-1322
+12240
 
-## Question 4 
+## Question 4
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+HTML (HyperText Markup Language) is the code that is used to structure a web page and its content.
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+Cascading Style Sheets or CSS is the language we use to style an HTML document.
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
-```css
-#h1 {
+<style>
```

Comment:
Hi Olena, good job! 
Can I ask, why you added <style></style> here?
Thank you!

Reply:
Hi [Farnooshmo](https://github.com/Farnooshmo)
I added <style></style> in HTML code and it  means that the document doesn't have separate css file. Correct answer can be 
h1{
}

Reply:
Dear Olena, Thank you for your response :)